monoloco/visuals/printer.py

```diff
@@ -165,38 +176,85 @@ def factory_axes(self, dic_out):
             axes.append(ax1)
         return figures, axes
 
-    def draw(self, figures, axes, image):
 
-        # whether to include instances that don't match the ground-truth
-        iterator = range(len(self.zz_pred)) if self.show_all else range(len(self.zz_gt))
-        if not iterator:
-            print("-" * 110 + '\n' + '! No instances detected' '\n' + '-' * 110)
+    def social_distance_front(self, axis, colors, annotations, dic_out):
+        sizes = [abs(self.centers[idx][1] - uv_s[1]*self.y_scale) / 1.5 for idx, uv_s in
+                 enumerate(self.uv_shoulders)]
 
-        # Draw the front figure
-        number = dict(flag=False, num=97)
-        if any(xx in self.output_types for xx in ['front', 'multi']):
-            number['flag'] = True  # add numbers
-            self.mpl_im0.set_data(image)
+        keypoint_sets, _ = get_pifpaf_outputs(annotations)
+        keypoint_painter = KeypointPainter(show_box=False, y_scale=self.y_scale)
+        r_h = 'none'
+        if 'raise_hand' in self.args.activities:
```

Comment:
Same here, regarding none. An option could be that the function directly accepts activities as argument and internally computes colors.